spec.bs

```diff
@@ -384,30 +395,43 @@ try {
 }
 </pre>
 
-# Dependencies # {#sctn-dependencies}
+# Terminology # {#sctn-terminology}
+
+: <dfn export>SPC Credential</dfn>
+:: A WebAuthn credential that can be used for behavior defined in this spec.
+    Can be used as a normal credential (e.g. for authentication flows), as well
+    as for SPC behavior. A [=Relying Party=] must explicitly opt-in to making an
+    SPC credential during credential creation.
+
+: <dfn>Steps to determine if a credential is SPC-enabled</dfn>
+:: An as-yet undefined process by which a user agent can, given a [=credential
+    ID=], determine if the credential represented by that ID is an [=SPC
+    Credential=].
 
-This specification relies on several other underlying specifications, listed
-below and in [[#index-defined-elsewhere]].
+    This allows the user agent to ensure that the powers of an SPC credential 
+    are only applied to credentials created as SPC-enabled.
```

Comment:
I think you can delete:

"This allows the user agent to ensure that the powers of an SPC credential are only applied to credentials created as SPC-enabled."

I think the intention is carried by the preceding sentence, and that the preceding sentence allows for the case of WebAuthn-in-1p-by-RP should we support that.

Reply:
Done.